packages/opencensus-core/src/exporters/console-exporter.ts

```diff
@@ -74,3 +71,29 @@ export class ConsoleExporter implements types.Exporter {
     });
   }
 }
+
+export class ConsoleStatsExporter implements types.StatsExporter {
+  /**
+   * Event called when a view is registred
+   * @param view registred view
+   * @param measure registred measure
+   */
+  onRegisterView(view: View, measure: Measure) {
+    console.log(`View registred: 
+    ${view},
```

Comment:
I would prefer that we not depend on whitespace for indentation in ``` `` ``` strings.

Reply:
I removed all the toString(). It wasn't necessary.